proxy/client.go

```
@@ -81,6 +88,11 @@ func (c *Client) newRequest(method, path string, body io.Reader) (*http.Request,
 //doRequest perform an HTTP request with context
 func (c *Client) doRequest(ctx context.Context, req *http.Request) (*http.Response, error) {
 	req = req.WithContext(ctx)
+
+	if os.Getenv("FAAS_CLI_DEBUG_HTTP") == "true" {
+		dump, _ := httputil.DumpRequest(req, true)
```

Don't swallow errors.